assets/js/theme/cart.js

```diff
@@ -158,9 +158,11 @@ export default class Cart extends PageManager {
 
         utils.api.productAttributes.configureInCart(itemId, options, (err, response) => {
             modal.updateContent(response.content);
-            const $productOptionsContainer = $('[data-product-attributes-wrapper]', this.$modal);
-            const modalBodyReservedHeight = $productOptionsContainer.outerHeight();
-            $productOptionsContainer.css('height', modalBodyReservedHeight);
+            this.$modal.on(ModalEvents.opened, () => {
```

Comment:
1. Looks like on every opening of the popup new event handler will be added and on the, for example third opening we will have three executions. Is it expected behaviour?
2. Do we cover a case when response come after popup opening already completed? Looks like now this code will not be executed

Reply:
thanks. moved it above the code so modal eventListener is added once.

Reply:
In that case event handler will be triggered when other content will be opened in this modal instance. It is better to add handler and remove it after execution